javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,20 @@
+/* exported findIndex */
+function findIndex(array, value) {
+  var i = array.length;
+  var j = 0;
+  var indexValue = -1;
+  if (i > 0) {
+    while (i > 0) {
+      if (array[j] === value) {
+        indexValue = j;
+        break;
+      } else {
+        i--;
+        j++;
+      }
+    }
+  } else {
+    indexValue = -1;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.